src/util/logging.py

```diff
@@ -17,30 +17,44 @@
 
 from util.context import Context
 
+log_call_count = dict()
+
+
+def log_multiline_message(message: Text, caller: Text):
+
+    # display logs called from one spot no more than 10 times, "None" means it was counted before
+    if caller != "None":
```

Comment:
you can combine this if and nested if with "AND"

Reply:
Then the else would need to be a separate if with two conditions, is that preferable?

Reply:
You are right, this way is better in such case, my mistake :)